test/test_type_promotion.py

```diff
@@ -693,10 +694,45 @@ def test_abs_complex_to_float(self, device, dtype):
         a = np.array(vals, dtype=numpy_dtype(dtype))
```

Comment:
can we also add a test for an empty complex tensor?

Reply:
You mean like `torch.abs(torch.empty((1,) dtype=torch.complex64))`? Sure we can. What might this catch?

Reply:
naah I meant `torch.abs(torch.tensor([], dtype=torch.cfloat))`

Reply:
Sure. I'll add a case for it ;)